src/DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -4111,7 +4111,7 @@ defaultproperties
     bingo_options(132)=(event="botordertrigger",desc="The Smuggler is whacked-out paranoid",max=1,missions=276)
 #ifdef injections
     bingo_options(133)=(event="IgnitedPawn",desc="Set %s people on fire",desc_singular="Set someone on fire",max=15)
-    bingo_options(134)=(event="GibbedPawn",desc="Blow up %s people",desc_singular="Blow someone up",max=15)
+    bingo_options(134)=(event="GibbedPawn",desc="Blow up %s people",desc_singular="Blow someone up",max=15,missions=57214)
```

Comment:
I don't think this should have a mission mask, since it's not really an explicitly included thing

```suggestion
    bingo_options(134)=(event="GibbedPawn",desc="Blow up %s people",desc_singular="Blow someone up",max=15)
```